### PR TITLE
cnap/core: secure enhancement for getting the model

### PR DESCRIPTION
**Pull request by dongx1x**
Add a get_model_securely function to enhance the security when getting a model, get the key from KBS (get the quote, do attestation), and decrypt the model.

### CODE REVIEW

File: cnap/core/modelprovider.py [hunk 1]
```diff
@@ -171,6 +180,38 @@ def get_model_data(self) -> bytes:
             self.get_model_info()
         return self._model_data
 
+    def get_model_securely(self, kbs_name, kbs_url, key_id) -> bytes:
+        """Get model securely.
+
+        This method is enhance the security to get the model, it shall run in Trusted Execution
```
Comment (leyao-daily) [inline cnap/core/modelprovider.py]: to enhance or enhancing?

File: cnap/core/modelprovider.py [hunk 2]
```diff
@@ -171,6 +180,38 @@ def get_model_data(self) -> bytes:
...
+            kbs_name(str): The name of the KBS.
+            kbs_url(str): The URL of the KBS.
+            key_id(str): The key ID to get the key to decrypt the model.
+
+        Returns:
+            bytes: The decrypted model data.
+
+        Raises:
+            ValueError: If model data or kbs input is invalid.
+            RuntimeError: If a the KBS can not be handled.
```
Comment (Hulongyin) [inline cnap/core/modelprovider.py]: If a or if the?

File: cnap/core/modelprovider.py (hunk 1 above)
Comment (dongx1x) [reply cnap/core/modelprovider.py]: Updated. Thanks.

File: cnap/core/modelprovider.py (hunk 2 above)
Comment (dongx1x) [reply cnap/core/modelprovider.py]: Updated. Thanks.

**Review by Hulongyin — APPROVED** [review]
LGTM. Thanks

**Pull request merged**